hugo-node/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN curl -L -o /tmp/node.tar.xz "https://nodejs.org/dist/${NODE_VERSION}/node-${
     && ln -s /usr/local/lib/nodejs/node-${NODE_VERSION}-linux-x64/bin/node /usr/bin/node \
     && ln -s /usr/local/lib/nodejs/node-${NODE_VERSION}-linux-x64/bin/npm /usr/bin/npm \
     && ln -s /usr/local/lib/nodejs/node-${NODE_VERSION}-linux-x64/bin/npx /usr/bin/npx \
-    && npm install -g yarn@1.22.19 \
```

Comment:
Not sure if there was a reason for this, but I think we should use the latest version instead of locking it to a stale release as we are doing with Drupal image.

Reply:
The latest version is 1.22.22 so it's does not change that often

Reply:
Based on the Jenkins failures that I am getting now, I think we need a different version of yarn for different versions of node.js.

I think we may need a ARG for yarn to work around this...

Reply:
Yarn version is not the issue AFAICT